backend/plugin/db/risingwave/risingwave.go

```diff
@@ -0,0 +1,493 @@
+// Package risingwave is the plugin for RisingWave driver.
+package risingwave
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"net"
+	"regexp"
+	"strings"
+	"time"
+
+	// Import pg driver.
+	// init() in pgx/v4/stdlib will register it's pgx driver.
+	"github.com/jackc/pgx/v4"
+	"github.com/jackc/pgx/v4/stdlib"
+	"github.com/pkg/errors"
+	"go.uber.org/multierr"
+	"go.uber.org/zap"
+	"golang.org/x/crypto/ssh"
+	"google.golang.org/protobuf/types/known/durationpb"
+
+	"github.com/bytebase/bytebase/backend/common"
+	"github.com/bytebase/bytebase/backend/common/log"
+	"github.com/bytebase/bytebase/backend/plugin/db"
+	"github.com/bytebase/bytebase/backend/plugin/db/util"
+	parser "github.com/bytebase/bytebase/backend/plugin/parser/sql"
+	storepb "github.com/bytebase/bytebase/proto/generated-go/store"
+	v1pb "github.com/bytebase/bytebase/proto/generated-go/v1"
+)
+
+var (
+	// ExcludedDatabaseList is the list of system or internal databases.
+	ExcludedDatabaseList = map[string]bool{
+		// Skip our internal "bytebase" database
+		"bytebase": true,
+		// Skip internal databases from cloud service providers
+		// see https://github.com/bytebase/bytebase/issues/30
+		// aws
+		"rdsadmin": true,
+		// gcp
+		"cloudsql":      true,
+		"cloudsqladmin": true,
```

Comment:
Are they saving for RisingWave?

Reply:
Copied, I'll review them carefully and remove some codes.